src/quantities.jl

```diff
@@ -56,13 +56,32 @@ end
 # ambiguity resolution
 //(x::AbstractQuantity, y::Complex) = Quantity(//(x.val, y), unit(x))
 
-for f in (:div, :fld, :cld)
-    @eval function ($f)(x::AbstractQuantity, y::AbstractQuantity)
-        z = uconvert(unit(y), x)        # TODO: use promote?
-        ($f)(z.val,y.val)
+for f in (:fld, :cld)
+    @eval begin
+        function ($f)(x::AbstractQuantity, y::AbstractQuantity)
+            z = uconvert(unit(y), x)        # TODO: use promote?
+            ($f)(z.val,y.val)
+        end
+
+        ($f)(x::Number, y::AbstractQuantity) = Quantity(($f)(x, ustrip(y)), unit(x) / unit(y))
+
+        ($f)(x::AbstractQuantity, y::Number) = Quantity(($f)(ustrip(x), y), unit(x))
     end
 end
 
+function div(x::AbstractQuantity, y::AbstractQuantity, r::RoundingMode=RoundToZero)
+    z = uconvert(unit(y), x)        # TODO: use promote?
+    div(z.val,y.val, r)
+end
```

Comment:
I'd do
```suggestion
function div(x::AbstractQuantity, y::AbstractQuantity, r...)
    z = uconvert(unit(y), x)        # TODO: use promote?
    div(z.val,y.val, r...)
end
```
so you don't even have to care to forcibly set a default value

Reply:
That sounds better than my idea.